analysis/org.eclipse.tracecompass.analysis.counters.core/src/org/eclipse/tracecompass/analysis/counters/core/aspects/CounterAspect.java

```diff
@@ -81,6 +81,7 @@ public CounterAspect(String fieldName, String label, CounterType type, Class<? e
      *
      * @return the groups
      */
```

Comment:
Now that it overrides, you can remove the Javadoc here, the one from interface will be used.

Reply:
not fixed